docs/reference/addons/velaux.md

```diff
@@ -173,6 +179,104 @@ spec:
 ```
 
 After deployed, let's get the root password from the secret `vela-system/velaux-db-mongodb`.
+</TabItem>
+
+<TabItem label="MySQL" value="mysql">
+
+```shell script
+vela addon enable velaux dbType=mysql dbURL=mysql://<MYSQL_USER>:<MYSQL_PASSWORD>@<MYSQL_HOST>:<MYSQL_PORT>/<MYSQL_DB_NAME>
+```
+> It's necessary to create the specified database in advance. You can find the relevant connection string parameters [here](https://github.com/go-sql-driver/mysql#parameters)
+
+You can also deploy the MySQL with this application configuration:
+
+> Your cluster must have a default storage class.
+
+```yaml
+apiVersion: core.oam.dev/v1beta1
+kind: Application
+metadata:
+  name: velaux-db
+  namespace: vela-system
+spec:
+  components:
+    - name: velaux-db
+      properties:
+        chart: mysql
+        repoType: helm
+        url: https://charts.bitnami.com/bitnami
+        values:
+          persistence:
+            size: 20Gi
+        version: 12.1.12
+      type: helm
+  policies:
+    - name: vela-system
+      properties:
+        clusters:
+          - local
+        namespace: vela-system
+      type: topology
+  workflow:
+    steps:
+      - name: vela-system
+        properties:
+          policies:
+            - vela-system
+        type: deploy
+```
+
+After deployed, let's get the root password from the secret `vela-system/velaux-db-mysql`.
+</TabItem>
+
+<TabItem label="PostgreSQL" value="postgresql">
+
+```shell script
+vela addon enable velaux dbType=postgresql dbURL=postgres://<POSTGRESQL_USER>:<POSTGRESQL_PASSWORD>@<POSTGRESQL_HOST>:<POSTGRESQL_PORT>/<POSTGRESQL_DB_NAME>
+```
+> It's necessary to create the specified database in advance.
+
+You can also deploy the PostgreSQL with this application configuration:
+
+> Your cluster must have a default storage class.
+
+```yaml
+apiVersion: core.oam.dev/v1beta1
+kind: Application
+metadata:
+  name: velaux-db
+  namespace: vela-system
+spec:
+  components:
+    - name: velaux-db
+      properties:
+        chart: postgresql
+        repoType: helm
+        url: https://charts.bitnami.com/bitnami
+        values:
+          persistence:
+            size: 20Gi
+        version: 12.1.12
+      type: helm
+  policies:
+    - name: vela-system
+      properties:
+        clusters:
+          - local
+        namespace: vela-system
+      type: topology
+  workflow:
+    steps:
+      - name: vela-system
+        properties:
+          policies:
+            - vela-system
+        type: deploy
+```
+
+After deployed, let's get the root password from the secret `vela-system/velaux-db-postgresql`.
```

Comment:
how to get the password? Give a shell command?

Reply:
Yes I think

Reply:
@NeerajGartia21 How is this going?

Reply:
I'm not able to fetch the secret in my local env. Can you please give it a try?